weblate/gitexport/utils.py

```diff
@@ -47,8 +47,10 @@ def find_git_http_backend():
         pass
 
     for path in GIT_PATHS:
-        name = os.path.join(path, "git-http-backend")
-        if os.path.exists(name):
-            find_git_http_backend.result = name
-            return name
+        names = ("git-http-backend", "git-http-backend.exe")
+        for name in names:
+            name = os.path.join(path, "git-http-backend")
```

Comment:
The name variable from the loop doesn't seem to be used, you probably had something like this in mind:

```suggestion
            name = os.path.join(path, name)
```

Reply:
Yes, good catch, my bad, I will fix once I can access a computer